src/components/date-picker/flatpickr-adapter/_flatpickr.scss

```diff
@@ -0,0 +1,888 @@
+// This file contains the original Flatpickr styles which are found in:
```

Comment:
While testing this a bit, I discovered that in the `datetime` picker, after you first click to select a date, and focus moves to the hour input field, if you want to change the date again, you have to click twice. The first click just blurs the hour input field, but doesn't select the date.

It's not caused by this PR (I've confirmed it's the same in the latest `next`), but I wanted to mention it here in case someone else runs into it too.

I'll create an issue for it.

Reply:
Issue: #1885 1885

Reply:
yeah I also saw that. it was so annoying. first I thought it was my changes, but it wasn't ;P Great that you made an issue